openquake/hazardlib/site.py

```diff
@@ -156,6 +156,7 @@ def _extract(array_or_float, indices):
 
     # other parameters
     'custom_site_id': numpy.uint32,
+    'eshm20_region': numpy.uint32
```

Comment:
.... I did not miss it in previous PR! I just missed this file :)

Reply:
Thanks Graeme for this additional explanation. I understand the way in which you use this parameter in your specific context but from a more general perspective this is just one additional - categorical - attribute for the site. Other models might also take advantage from it hence I think it's good to keep what you did general.

Reply:
OK, now changed to `region`